addons/middle-click-popup/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Insert blocks by name",
```

I thought of a name...
```suggestion
  "name": "Block summoning",
```
Feel free to skip this - it's somewhat vague for the name, but it's fun and the description might cover it.

---

I like how it's fun, but we probably shouldn't use this.

---

> I like how it's fun, but we probably shouldn't use this.

I thought so. Most of the other addons are more descriptive than app-like in the names.